common/lib/capa/capa/responsetypes.py

```diff
@@ -1097,6 +1097,9 @@ def get_score(self, student_answers):
                 cmap.set(aid, 'correct')
             else:
                 cmap.set(aid, 'incorrect')
+            answer_variable = self.get_student_answer_variable_name(student_answers, aid)
```

Comment:
@symbolist Yeah we can do that but ```evaluate_answers()``` is called for every response type and we need ```answervariable``` in ```OptionResponse``` only that's why i'm doing this in ```OptionResponse```'s ```get_score()```.